ironfist/installer/map_xml.xsd

```diff
@@ -81,14 +81,24 @@
   </xs:complexType>
 
   <xs:complexType name="mapVariable_t">
+    <xs:sequence>
+      <xs:element name="array" minOccurs="0" maxOccurs="unbounded">
+        <xs:complexType>
+          <xs:attribute name="key" type="xs:string"/>
+          <xs:attribute name="value" type="xs:string"/>

```

Comment:
This needs to be recursive. You can have tables be values in tables. Important for having e.g.: 2D arrays.